INSTALLING.md

```diff
@@ -157,6 +157,24 @@ sudo apt-get install cpufrequtils
 sudo cpupower frequency-set -g performance
 ```
 
+#### Option 3: From Docker Release Image
+
+Download the latest Docker release from our [Docker registry](https://github.com/orgs/tenstorrent/packages?q=tt-metalium-ubuntu&tab=packages&q=tt-metalium-ubuntu-20.04-amd64-dev) page for
```

Comment:
If these are release images, should it not be as a release amd64-release?

Reply:
Good catch - I was testing with dev but this should be corrected to release